fastlane/lib/fastlane/actions/carthage.rb

```diff
@@ -42,6 +43,10 @@ def self.validate(params)
         if command_name != "archive" && params[:output]
           UI.user_error!("Output option is available only for 'archive' command.")
         end
+
+        unless %w(build bootstrap update).include?(command_name)
+          params[:log_path] = nil
```

Comment:
Can we call `UI.user_error!(` instead so users know / aren't surprised why its getting cleared 🙃